example/spec/emsReport/emsIngestReportSpec.js

```diff
@@ -0,0 +1,240 @@
+const fs = require('fs-extra');
```

Comment:
Could these tests go in the `spec/parallel` folder, especially since they're particularly long running?

Reply:
EMS report and reconciliation report tests now both use the same collection, and collection name can't be modified (The collection has to be in CMR for reconciliation report, and the collection has to be in EMS for report).  Maybe I can get a different collection later for EMS report.

Reply:
Put CreateReconciliationReport and EmsReport tests to spec/parallel